### PR TITLE
Reflect renaming of repository

### PR DESCRIPTION
**Pull request by aravindhp**
The repository name has changed from "windows-machine-config-operator" to "windows-machine-config-bootstrapper" to indicate this repo is related to WMCB and not the operator. A new repo called "windows-machine-config-operator" will be created to house the operator code.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,20 +11,23 @@ replace (
 )
 
 require (
+	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2 // indirect
```
Comment (suhanime) [inline go.mod]: Why are the dependencies for `github.com/Azure/go-autorest/autorest/azure/auth`, `github.com/masterzen/winrm` and `github.com/pkg/sftp` added?

Comment (aravindhp) [reply go.mod]: Thanks for catching this. I have fixed it.

**Comment by aravindhp** [comment]
Both `e2e-wsu` and `e2e-wmcb` tests are failing as expected:
```
go: github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer@v0.0.0-20200130180731-c1f47e5c73f3: parsing go.mod: unexpected module path "github.com/openshift/windows-machine-config-operator/tools/windows-node-installer"
```
Will override them for the merge to go through.

**Pull request merged**